forms.py

```diff
@@ -28,31 +28,31 @@ class SubmitJob(Form):
                                        "of chromosome. Note: Position is 0-based",
                            validators=[Optional()])
     # OR
-    upstream_fasta = FileField('Upstream Sequence',
-                               description="FASTA file with upstream sequence.",
+    upstream_fasta = MultipleFileField('Upstream Sequence',
+                               description="FASTA files with upstream sequence.",
                                validators=[
                                    Optional()
                                ])
-    downstream_fasta = FileField('Downstream Sequence',
-                                 description="FASTA file with downstream sequence.",
+    downstream_fasta = MultipleFileField('Downstream Sequence',
+                                 description="FASTA files with downstream sequence.",
                                  validators=[
                                      Optional()
                                  ])
 
     # Uploads
-    in_fasta = FileField('Inserted Sequence (FASTA)',
-                         description="New sequence to be inserted into reference genome.",
+    in_fasta = MultipleFileField('Inserted Sequence (FASTA)',
+                         description="New sequences to be inserted into reference genome.",
                          validators=[
                              Optional(),
                              FileAllowed([ALLOWED_EXTENSIONS], 'Invalid File Type'),
                              FileRequired()
                          ])
-    in_gff = FileField('Inserted Reference (gff3 or gtf)',
-                       description="GFF file describing new FASTA sequence to be inserted.",
+    in_gff = MultipleFileField('Inserted Reference (gff3 or gtf)',
+                       description="GFF files describing new FASTA sequence to be inserted.",
```

Comment:
Change to `description="GFF file(s) describing new FASTA sequence to be inserted.",`